components/private/Math/src/MDCMath.h

```diff
@@ -131,6 +131,22 @@ static inline CGFloat MDCSqrt(CGFloat value) {
 #endif
 }
 
+static inline CGFloat MDCCeilScaled(CGFloat value, CGFloat scale) {
```

Comment:
Please add documentation here and in `MDCFloorScaled` to describe how these work, particularly with different values of `scale`.